kerl

```diff
@@ -1922,9 +1924,41 @@ maybe_remove() {
 }
 
 list_print() {
-    if [ -f "$KERL_BASE_DIR/otp_$1" ]; then
-        if [ "$(\wc -l "$KERL_BASE_DIR/otp_$1")" != '0' ]; then
-            cat "$KERL_BASE_DIR/otp_$1"
+    list_type=$1 # releases | builds | installations
+    maybe_all=$2
+
+    list=$KERL_BASE_DIR/otp_${list_type}
+    if [ -f "$list" ]; then
+        if [ "$(\wc -l "$list")" != '0' ]; then
+            if [ "${list_type}" = releases ] && [ "$maybe_all" != all ]; then
+                awk -v oldest_listed="$OLDEST_OTP_LISTED" \
+                    -v oldest_supported="$OLDEST_OTP_SUPPORTED" \
+                    -v last_major="$OLDEST_OTP_LISTED" '
+                {
+                    full_version=$0
+                    split(full_version, version_components, ".")
+                    major=version_components[1]
+                    if (major >= oldest_listed && !/-rc/ && !/^R/) {
+                        if (major > last_major) {
+                            if (last_major >= oldest_supported) {
+                                printf "* "
+                            }
+
+                            printf "%s\n",last_full_version
+                        }
+                        last_major=major
+                        last_full_version=full_version
+                    }
+                }
+                END {
+                    if (!/-rc/) {
+                        printf "* %s\n",$0
```

Comment:
Put the asterisk behind the version tag
```suggestion
                        printf "%s *\n",$0
```

Reply:
I'll accept the suggestion, but also change the README at the same time, so I can't commit from this one 😄

Reply:
5ae4131